dcrjson/walletsvrcmds.go

```diff
@@ -786,6 +786,40 @@ func NewPurchaseTicketCmd(fromAccount string, spendLimit float64, minConf *int,
 	}
 }
 
+// CreateUnsignedTicketCmd is a type handling custom marshaling and
+// unmarshaling of createunsignedticket JSON RPC commands.
+type CreateUnsignedTicketCmd struct {
+	FromAccount   string
+	SpendLimit    float64 // In Coins
+	MinConf       *int    `jsonrpcdefault:"1"`
+	TicketAddress *string
+	NumTickets    *int
```

Comment:
I would suggest defining a `jsonrpcdefault:"1"` here to better document the semantics.